src/content/ui/interactivity/focus.md

```diff
@@ -3,179 +3,226 @@ title: Understanding Flutter's keyboard focus system
+since there is _always_ a primary focus. When a node calls `unfocus()`,
+focus can pass to one of two nodes.
+It can pass to either the nearest `FocusScopeNode`
+or a node in the same scope that had focus earlier.
+
+Which node Flutter chooses depends upon the `disposition` argument
+you give to `unfocus()`.
+When calling `unfocus()`, you can set the `disposition` argument.
+It allows two modes for passing focus:
+
+* [`UnfocusDisposition.scope`][]:
+  The argument defaults to this option.
+  It gives the focus to the nearest parent focus scope.
+  If then something moves the focus to the next node with `FocusNode.nextFocus`,
+  the app starts with the "first" focusable item in the scope.
+
+* `UnfocusDisposition.previouslyFocusedChild`:
+  The `previouslyFocusedChild` disposition searches the scope to find the
+  most recently focused child and requests focus on it.
+  If no child had focus before, this disposition works like `scope`.
+
+To better control where the focus passes when you remove it from a node,
+you have two options.
+You can choose to set focus to another node using `requestFocus()`
+or use the focus traversal mechanism.
+To find another node, this mechanism can use the `focusInDirection`,
 `nextFocus`, or `previousFocus` methods on `FocusNode`.
 
-When calling `unfocus()`, the `disposition` argument allows two modes for
-unfocusing: [`UnfocusDisposition.scope`][] and
-`UnfocusDisposition.previouslyFocusedChild`. The default is `scope`, which gives
-the focus to the nearest parent focus scope. This means that if the focus is
-thereafter moved to the next node with `FocusNode.nextFocus`, it starts with the
-"first" focusable item in the scope.
-
-The `previouslyFocusedChild` disposition will search the scope to find the
-previously focused child and request focus on it. If there is no previously
-focused child, it is equivalent to `scope`.
-
-:::secondary Beware
-If there is no other scope, then focus moves to the root scope node of
-the focus system, `FocusManager.rootScope`. This is generally not desirable, as
-the root scope doesn't have a `context` for the framework to determine which
-node should be focused next. If you find that your application suddenly loses
-the ability to navigate by using focus traversal, this is probably what has
-happened.  To fix it, add a `FocusScope` as an ancestor to the focus node that
-is requesting the unfocus. The `WidgetsApp` (from which `MaterialApp` and
-`CupertinoApp` are derived) has its own `FocusScope`, so this should not be an
-issue if you are using those.
-:::
+## Allow a control to receive focus
+
+The `Focus` widget owns and manages a focus node.
+It performs the following functions as the workhorse of the focus system:
 
-## Focus widget
+* Attaches and detaches the focus node it owns from the focus tree
+* Manages the attributes and callbacks of the focus node
+* Enables discovery of focus nodes attached to the widget
+  tree through its static functions
```

Comment:
Why did you replace this text with a bulleted list? Particularly following a header, it should prose, not a list. It's much harder to parse this way.